pkg/scheduler/scheduling_application.go

```diff
@@ -624,13 +663,43 @@ func (sa *SchedulingApplication) tryNodes(ask *schedulingAllocationAsk, nodeIter
 		}
 		// return allocation proposal and mark it as a reservation
 		alloc := newSchedulingAllocation(ask, nodeToReserve.NodeID)
+		err := events.EmitReserveEvent(ask.AskProto.AllocationKey, ask.ApplicationID, nodeToReserve.NodeID)
+		if err != nil {
+			log.Logger().Debug("could not emit reserve events to shim",
+				zap.String("allocationKey", ask.AskProto.AllocationKey),
+				zap.String("appID", ask.ApplicationID),
+				zap.String("nodeID", nodeToReserve.NodeID),
+				zap.Error(err))
+		}
 		alloc.result = reserved
 		return alloc
 	}
 	// ask does not fit, skip to next ask
 	return nil
 }
 
+func EmitAllocatedReservedEvents(allocKey, appID, nodeID string) {
+	err := events.EmitAllocatedReservedEvent(allocKey, appID, nodeID)
+	if err != nil {
+		log.Logger().Warn("could not emit allocatedReserved events to shim",
```

Comment:
be consistent when using logging level: seems to change from WARN to DEBUG for similar errors.

Reply:
You're right. Changed the WARNs to DEBUGs.